src/System.Collections.NonGeneric/tests/Performance/System.Collections.NonGeneric.Performance.Tests.csproj

```diff
@@ -20,6 +20,10 @@
     <ProjectReference Include="..\..\pkg\System.Collections.NonGeneric.pkgproj">
       <Name>System.Collections.NonGeneric</Name>
     </ProjectReference>
+    <!-- Do not remove this P2P reference since part of the implementation of NonGeneric has moved to Runtime.Extensions -->
+    <ProjectReference Include="..\..\..\System.Runtime.Extensions\pkg\System.Runtime.Extensions.pkgproj" />
+    <!-- ToDo: Remove this P2P reference once packages are updated -->
```

Comment:
> Why does it need S.Security.Permissions reference?

I assume it's because the test projects now automatically reference everything, which means any time you move a type from assembly A to assembly B, you need to include P2P references for both A and B so that the test project doesn't see the type duplicated between the old package of A and the new assembly of B. I've run into that recently any time I've moved a type.